addons/website/static/src/snippets/s_process_steps/000.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="website.s_process_steps.defs">
+        <defs>
```

Comment:
Don't indent the first level of XML files, just add blank lines between the templates (to separate from `<templates`)